Magic/1. The Praeter.md

```diff
@@ -0,0 +1,24 @@
+# 1. About Magic: the Praeter
+
+## The praeter
+The praeter is a part, a dimension of the universe, whose mechanics and relation to the physical universe are not well understood. His access allows trained casters to perform magic. They can either enter the praeter directly or use sources of power displayed around the world.
+
+People aren't born with some gift to perform magic. Although anyone could enter it with the help of a source of power, the contact with the praeter is very exhausting and will basically faint any uninitiated.
+
+## The direct way
+Accessing the praeter directly requires years and years of training and a lot of energy. Thereby practically nobody can perform magic this way. Adepts of this methode are extremely powerful.
+
+It is said that some of them could even use the praeter to keep their spirit immortal and even come back in the real world as a newborn. They are sometimes perceived as divine or even as gods by certain folks.
```

Comment:
Entering the preater still sounds a bit confused to me. Should we keep it like this to make it a somehow secret technique ? Also, I am not sure to like the idea of a completely separated plan, where people enter and leave. What do you think of a more mixed with reality thing ?